src/app/ubs/ubs-user/ubs-user-profile-page/ubs-user-profile-page.component.ts

```diff
@@ -193,18 +211,43 @@ export class UbsUserProfilePageComponent implements OnInit {
           this.userProfile = this.composeFormData(res);
           this.userProfile.recipientEmail = this.userForm.value.recipientEmail;
           this.userProfile.alternateEmail = this.userForm.value.alternateEmail;
+          this.userProfile.telegramIsChecked = this.userForm.value.telegramIsChecked;
+          this.userProfile.viberIsChecked = this.userForm.value.viberIsChecked;
         },
         (err: Error) => {
           this.isFetching = false;
           this.snackBar.openSnackBar('ubs-client-profile.error-message');
         }
       );
       this.alternativeEmailDisplay = false;
+      this.redirectToMessengers();
     } else {
       this.isEditing = true;
     }
   }
 
+  redirectToMessengers() {
```

Comment:
update this method
redirectToMessengers() {
if (this.telegramNotification) {
        this.goToTelegramUrl();
      }
      if (this.viberNotification) {
        this.goToViberUrl();
      }
}

Reply:
Yeah,thanks